src/HotelCalifornia/Exec.hs

```diff
@@ -7,11 +7,14 @@ import qualified Data.Char as Char
 import Data.List.NonEmpty (NonEmpty(..))
 import qualified Data.List.NonEmpty as NEL
 import Data.Maybe (fromMaybe)
-import Data.Text (Text)
+import Data.HashMap.Strict (HashMap)
+import qualified Data.HashMap.Strict as HashMap
 import qualified Data.Text as Text
+import Data.Text (Text, breakOn, pack)
```

Comment:
```suggestion
import Data.Text (Text)
```

I'd prefer to see `Text` used qualified with the exception of the name of the type